app/pages/project/index.cjsx

```diff
@@ -130,9 +134,9 @@ ProjectPage = React.createClass
         @getSelectedWorkflow(project, @props.preferences)
 
   getSelectedWorkflow: (project, preferences) ->
-    # preference user selected workflow, then project owner set workflow, then default workflow
+    # preference workflow query, then user selected workflow, then project owner set workflow, then default workflow
```

Comment:
Yeah there are comments all over the classifier, so I left a few myself. I could go either way on it.